src/KubeOps.Operator/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("KubeOps.Operator.Tests")]
```

Comment:
This is not needed, as the internals visible to is provided by the `Directory.Build.props` file in the `src` folder.